handlers/people-handler/two_people.py

```diff
@@ -0,0 +1,155 @@
+# This file will generate a rendering if the image contains two people
+
+import logging
+logging.basicConfig(level=logging.NOTSET)
+
+
+def common_check(inanimate1, inanimate2):
+    check_common = []
+    for i in range(len(inanimate1)):
+        for j in range(len(inanimate2)):
+            if (inanimate1[i]["ID"] == inanimate2[j]["ID"]):
+                check_common.append(inanimate1[i])
+                break
+    return check_common
+
+# calculate the dominant emotion
+# among the two individuals present in the image
+
+
+def calculate_dominant_emotion_for_two(happy, sad, neutral):
+    if (happy == 2):
+        return "happy expression "
+    if (sad == 2):
+        return "sad expression "
+    if (neutral == 2):
+        return "serious expression "
+    if (happy == 1 and neutral == 1):
+        return "happy  expressions "
+    if (neutral == 1 and sad == 1):
+        return "sad expressions "
+    if (happy == 1):
```

Comment:
If one is happy and one is sad, "happy" is returned due to the ordering of the if statements. Is this a conscious choice? Why does "happy" dominate "sad" if both are present?